core/tests/Test.Microsoft.Identity.Core.UIAutomation/Test.Microsoft.Identity.Core.UIAutomation.csproj

```diff
@@ -32,7 +32,7 @@
     <PackageReference Include="NUnit" Version="2.6.4" />
     <PackageReference Include="Xamarin.UITest" Version="2.2.4" />
     <PackageReference Include="NUnitTestAdapter" Version="2.1.1" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.*" />
+    <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
```

Comment:
Same comment as before - please leave as is unless you have a sepcific reason to use that old version of the lib. Currently we have no build warnings except for the "Cannot build without a MAC" false positive, so we should aim to mantain 0 warnings.

Reply:
Hard requirement by automation framework to use this version